ICE.js

```diff
@@ -994,11 +994,17 @@ var ICE = {
         if(!method)
             method = "POST";
 
+        if(settings) {
+            if(settings["cache"] === false && method === "GET") {
+                url += "&_=" + Date.now();
```

Comment:
If url doesn't have a query string at all, it will break it, I guess: `http://example.com` -> `http://example.com&_=1234567.00`